packages/db/src/models/execution.ts

```diff
@@ -34,6 +34,25 @@ class ExecutionPgModel extends BasePgModel<PostgresExecution, PostgresExecutionR
       .merge()
       .returning('cumulus_id');
   }
+
+  /**
+   * Get executions from the execution cumulus_id
+   *
+   * @param {Knex | Knex.Transaction} knexOrTrx -
+   *  DB client or transaction
+   * @param {Array<number>} executionCumulusIds -
+   * single execution cumulus_id or array of exeuction cumulus_ids
+   * @returns {Promise<Array<number>>} An array of exeuctions
+   */
+  async getExecutionsFromCumulusIds(
```

Comment:
```suggestion
  async searchByCumulusIds(
```

Very subjective. Take it or leave it

Reply:
Need a unit test here though

Reply:
Yeah I like the search phrasing for these functions so made those changes throughout